components/workflow-mgt/org.wso2.carbon.identity.workflow.impl/pom.xml

```diff
@@ -131,7 +131,8 @@
                             org.wso2.carbon.core.util; version="${carbon.kernel.package.import.version.range}",
                             org.wso2.carbon.stratos.common.*; version="${carbon.commons.imp.pkg.version}",
                             org.wso2.carbon.identity.workflow.mgt.*; version="${carbon.identity.package.import.version.range}",
-                            org.wso2.carbon.humantask.deployer; resolution:=optional
+                            org.wso2.carbon.humantask.deployer; resolution:=optional,
+                            org.apache.axiom.om.util
```

Comment:
Thanks for pointing, Fixed with https://github.com/wso2/carbon-identity/pull/1675